src/Views/WhatsNewPage.xaml

```diff
@@ -33,7 +33,7 @@
 
     <ScrollViewer HorizontalScrollMode="Disabled" VerticalScrollMode="Auto" MaxWidth="{ThemeResource MaxPageBackgroundWidth}">
         <RelativePanel>
-            <Grid>
+            <Grid CornerRadius="8">
```

Comment:
nit: If we want this solution for all pages, consider adding the corner radius in the ShellPage.xaml
```xaml
<Grid CornerRadius="{ThemeResource OverlayCornerRadius}" Grid.Row="1">
    <Frame x:Name="NavigationFrame" />
</Grid>
```